questionnaire-1-egifter.md

```diff
@@ -0,0 +1,41 @@
+# Egifter
+
+Egifter is a crypto-friendly gift card provider that claims to allow companies to top-up their accounts in crypto and send rewards in fiat https://www.egifter.com/
+
+After the 1st application, they requested more information because Ubiquity is not registered in the US.
+The questionnaire has been sent and is under verification now to receive approval to use our account.
+
+### What kind of business are you running? What is your business model?
+
+Ubiquity is a startup building a freelance marketplace with AI.
+Freelancers are individual software developers located across the globe who contribute to Open Source projects on GitHub.
+The Ubiquity freelance marketplace supports public good with the contribution to Open Source.
+
+Our business model is under research now. 
+Preliminary, we will charge from the Open Source projects. 
```

Comment:
Perhaps it makes sense to say that we plan to charge for a monthly service to use our SaaS product to recruit and manage remote development teams.